File: Service_eHospital_DongNai_A_NSTL/Implement/NS_DM_NGHENGHIEPService.cs
namespace Service_eHospital_DongNai_A_NSTL.Implement
{
    public class NS_DM_NGHENGHIEPService : BaseService<NS_DM_NGHENGHIEP, INS_DM_NGHENGHIEPRepository>
    , INS_DM_NGHENGHIEPService
    {
    private readonly INS_DM_NGHENGHIEPRepository _NS_DM_NGHENGHIEPRepository;
    public NS_DM_NGHENGHIEPService(INS_DM_NGHENGHIEPRepository NS_DM_NGHENGHIEPRepository) : base(NS_DM_NGHENGHIEPRepository)
    {
    _NS_DM_NGHENGHIEPRepository = NS_DM_NGHENGHIEPRepository;
    }
    }
    }

File: Report/src/app/shared/eHospital_DongNai_A_NSTL/NS_DM_NGHENGHIEP.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { NS_DM_NGHENGHIEP } from './NS_DM_NGHENGHIEP.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class NS_DM_NGHENGHIEPService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'MaNgheNghiep', 'TenNgheNghiep'];

    List: NS_DM_NGHENGHIEP[] | undefined;
    ListFilter: NS_DM_NGHENGHIEP[] | undefined;
    FormData!: NS_DM_NGHENGHIEP;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "NS_DM_NGHENGHIEP";
    }
}

